### ISSUE TITLE
SQL autocompletion support

### ISSUE THREAD

**Issue by sunker** [grafana/redshift-datasource#108]
We should add support for syntax highlighting and autcomplete in the Redshift data source. The work 

### Tasks
- [x] #150
- [x] #153
- [x] #155
- [x]  Add default completion item provider to experimental package in the case standard sql is used https://github.com/grafana/grafana-experimental/pull/42
- [x] Support schema in table identifiers https://github.com/grafana/grafana-experimental/pull/43
- [ ]  #168
- [ ] #170
- [ ] #172
- [ ] Always render new code editor and remove feature flag from Grafana

There are a few things that I don't like about the SQL Editor api in grafana/experimental. For example, functions, operators and supported macros are currently being specified in the autocompletion provider and not in the language. This causes a discrepancy between the syntax highlighting support and the autocompletion support. I'll raise an issue about this in the experimental package so that it can be discusses with the BI squad.

**Comment by sunker** [comment]
Temporary putting this issue on hold so that I can focus on Grafana 9 critical issues instead

**Comment by sarahzinger** [comment]
@sunker is it cool if I move this to backlog?

**Comment by sunker** [comment]
Yes sure!

**Comment by vickyyyyyyy** [comment]
The Timestream and Athena tickets should be picked up once this is done
